main.go

```diff
@@ -42,7 +42,8 @@ func main() {
 func run() error {
 	flag.Parse()
 	now := time.Now().UTC()
-	gc := git.Config{}
+	gc := &git.Config{}
```

Comment:
Why do we need this to pass this by reference?

Reply:
I think either is fine, but it is better to reduce the use of memory.